android/src/main/java/com/oney/WebRTCModule/CameraCaptureController.java

```diff
@@ -73,6 +107,7 @@ public void onCameraSwitchError(String s) {
                         handler.onSwitchCameraDone(facingMode());
                     }
                 });
+                return;
```

Comment:
oh yikes good catch! what was the effect? just an unnecessary switch to the same device? (so, just a bit of wasted work but nothing catastrophic?)